web3_proxy/src/app/mod.rs

```diff
@@ -749,12 +823,25 @@ impl Web3ProxyApp {
 
             app_handles.push(config_handle);
         }
+        // =======
+        //         if important_background_handles.is_empty() {
+        //             info!("no important background handles");
+        //
+        //             let f = tokio::spawn(async move {
+        //                 let _ = background_shutdown_receiver.recv().await;
+        //
+        //                 Ok(())
+        //             });
+        //
+        //             important_background_handles.push(f);
```

Comment:
we still need this. its necessary if the app is run with influx/mysql disabled.

Reply:
Please double-check in latest commit